scripts/extract_download_logs.py

```diff
@@ -57,7 +80,32 @@
     "rp_start",
     "rp_end",
 ]
-OUTPUT_FILENAME = options.filename
+OUTPUT_FILENAME = args.filename
+
+
+def send_notify(
+    from_date: datetime.datetime,
+    to_date: datetime.datetime,
+    file: StringIO,
```

Comment:
This should be `BytesIO`. Probably best not to call this `file` as well as more of an in-memory buffer

Reply:
fixed in: d2f0fba